client/blocks/signup-form/crowdsignal.jsx

```diff
@@ -132,22 +132,26 @@ class CrowdsignalSignupForm extends Component {
 									>
 										{ translate( 'Create a WordPress.com Account' ) }
 									</FormButton>
+
+									<p className="signup-form__crowdsignal-learn-more">
+										{ translate( 'Why WordPress.com? {{a}}Learn more.{{/a}}', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Why WordPress.com? {{a}}Learn more{{/a}}.' )` *ES Score: 6*